src/components/Home/Home.jsx

```diff
@@ -20,8 +20,15 @@ const Home = ({ strings }) => (
     <Why />
     <Sponsors />
     <BottomTextDiv>
-      {strings.home_background_by}
-      <a href="//www.artstation.com/artist/mikeazevedo" target="_blank" rel="noopener noreferrer"> Mike Azevedo</a>
+      <span id="bg-image-description">{strings.home_background_by}</span>
+      <a 
+        href="//www.artstation.com/artist/mikeazevedo" 
+        target="_blank" 
+        rel="noopener noreferrer" 
+        aria-describedby="bg-image-description" 
+        aria-label="Mike Azevedo on artstation.com"
+      > Mike Azevedo
+      </a> 
```

Comment:
Even though screen reader users may not experience the background image the same way visual users do, because there is a focusable link to the artist's profile, more context should be provided as to why a screen reader is dictating the link to the user. Currently screen readers will only read `Mike Azevedo`, leaving it up to the user to try and understand the meaning of the link. Adding an `aria-describedby` provides a little more meaning to screen reader users.